src/HotChocolate/ElasticSearch/test/Data.ElasticSearch.Tests/__snapshots__/StringPrefixTests.ElasticSearch_MultipleField_OneNegated.snap

```diff
@@ -0,0 +1,5 @@
+﻿{
```

Comment:
query?

Reply:
should be found under `__snapshots__/StringPrefixTests.ElasticSearch_MultipleField_OneNegated_query.snap`.